tests/helpers/model.py

```diff
@@ -23,11 +23,11 @@
 from pytorch_lightning.callbacks import ModelCheckpoint
 
 from anomalib.config import get_configurable_parameters, update_nncf_config
-from anomalib.core.callbacks import get_callbacks
-from anomalib.core.callbacks.visualizer_callback import VisualizerCallback
-from anomalib.core.model.anomaly_module import AnomalyModule
 from anomalib.data import get_datamodule
 from anomalib.models import get_model
+from anomalib.models.components import AnomalyModule
+from anomalib.utils.callbacks import get_callbacks
```

Comment:
Maybe we can do `from anomalib.utils.callbacks import get_callbacks, VisualizerCallback`